src/main/java/entity/PlayerStatistic.java

```diff
@@ -0,0 +1,254 @@
+        this.steals = steals;
+        this.team = team;
+        this.threePointAttempts = threePointAttempts;
+        this.threePointMade = threePointMade;
+        this.threePointPercentage = threePointPercentage;
+        this.totalRebounds = totalRebounds;
+        this.turnovers = turnovers;
+        this.twoPointAttempts = twoPointAttempts;
+        this.twoPointMade = twoPointMade;
+        this.twoPointPercentage = twoPointPercentage;
+    }
+
+    // Getters for all fields
+    public int getAge() {
+        return age;
+    }
+
+    public int getAssists() {
+        return assists;
+    }
+
+    public int getBlocks() {
+        return blocks;
+    }
+
+    public int getDefensiveRebounds() {
+        return defensiveRebounds;
+    }
+
+    public double getEffectiveFieldGoalPercentage() {
+        return effectiveFieldGoalPercentage;
+    }
+
+    public int getFieldGoalsAttempted() {
+        return fieldGoalsAttempted;
+    }
+
+    public int getFieldGoalsMade() {
+        return fieldGoalsMade;
+    }
+
+    public double getFieldGoalPercentage() {
+        return fieldGoalPercentage;
+    }
+
+    public int getFreeThrowsMade() {
+        return freeThrowsMade;
+    }
+
+    public int getFreeThrowsAttempted() {
+        return freeThrowsAttempted;
+    }
+
+    public double getFreeThrowPercentage() {
+        return freeThrowPercentage;
+    }
+
+    public int getGamesPlayed() {
+        return gamesPlayed;
+    }
+
+    public int getGamesStarted() {
+        return gamesStarted;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public int getMinutesPlayed() {
+        return minutesPlayed;
+    }
+
+    public int getOffensiveRebounds() {
+        return offensiveRebounds;
+    }
+
+    public int getPersonalFouls() {
+        return personalFouls;
+    }
+
+    public String getPlayerId() {
+        return playerId;
+    }
+
+    public String getPlayerName() {
+        return playerName;
+    }
+
+    public int getPoints() {
+        return points;
+    }
+
+    public String getPosition() {
+        return position;
+    }
+
+    public int getSeason() {
+        return season;
+    }
+
+    public int getSteals() {
+        return steals;
+    }
+
+    public String getTeam() {
+        return team;
+    }
+
+    public int getThreePointAttempts() {
+        return threePointAttempts;
+    }
+
+    public int getThreePointMade() {
+        return threePointMade;
+    }
+
+    public double getThreePointPercentage() {
+        return threePointPercentage;
+    }
+
+    public int getTotalRebounds() {
+        return totalRebounds;
+    }
+
+    public int getTurnovers() {
+        return turnovers;
+    }
+
+    public int getTwoPointAttempts() {
+        return twoPointAttempts;
+    }
+
+    public int getTwoPointMade() {
+        return twoPointMade;
+    }
+
+    public double getTwoPointPercentage() {
+        return twoPointPercentage;
+    }
+
+    public String formatPlayerStats() {
```

Comment:
Ensure all relevant statistics are included for clarity.

Reply:
Yes I fixed that